### PR TITLE
Q2D example stripline

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: examples/05-Q3D/Q2D_Example_Stripline.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+from pyaedt import Q2d
+from pyaedt.generic.general_methods import generate_unique_name
+
+###############################################################################
+# Launch AEDT in Non-Graphical Mode
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# You can change the Boolean parameter ``NonGraphical`` to ``False`` to launch
+# AEDT in graphical mode.
+
+NonGraphical = True
```
Comment (akaszynski) [inline examples/05-Q3D/Q2D_Example_Stripline.py:19]: Avoid PascalCase for parameter names.

File: examples/05-Q3D/Q2D_Example_Stripline.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
+"""
+2D Extractor: Stripline Analysis
+---------------
```
Comment (akaszynski) [inline examples/05-Q3D/Q2D_Example_Stripline.py:3]:
```suggestion
2D Extractor: Stripline Analysis
--------------------------------
```

File: examples/05-Q3D/Q2D_Example_Stripline.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+home = os.path.expanduser("~")
+workdir = os.path.join(home, "Downloads", "pyaedt_example")
+project_path = os.path.join(workdir, generate_unique_name("pyaedt_q2d_example") + ".aedt")
+
+###############################################################################
+# Launch AEDT and Q2D
+# ~~~~~~~~~~~~~~~~~~~
+# This example launches AEDT 2022.1 in graphical mode.
+
+# This example use SI units.
```
Comment (MaxJPRey) [inline examples/05-Q3D/Q2D_Example_Stripline.py:30]:
```suggestion
# This example uses SI units.
```

File: examples/05-Q3D/Q2D_Example_Stripline.py [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+q.assign_single_conductor(
+    name=obj.name, target_objects=[obj], conductor_type="SignalLine", solve_option="SolveOnBoundary", unit="mm"
+)
+
+obj = q.modeler.get_object_from_name("signal_n")
+q.assign_single_conductor(
+    name=obj.name, target_objects=[obj], conductor_type="SignalLine", solve_option="SolveOnBoundary", unit="mm"
+)
+
+# ~~~~~~~~~~~~~~~~~
```
Comment (MaxJPRey) [inline examples/05-Q3D/Q2D_Example_Stripline.py:139]: @ring630 Maybe we can get ride of it here, but it is up to you.
```suggestion
```

Comment (ring630) [reply examples/05-Q3D/Q2D_Example_Stripline.py:139]: Sure. I was copying the coding style from the other example.

File: examples/05-Q3D/Q2D_Example_Stripline.py (hunk 1 above)
Comment (ring630) [reply examples/05-Q3D/Q2D_Example_Stripline.py:19]: Done

**Review by maxcapodi78 — APPROVED** [review]
LGTM

**Pull request merged**